formatter/formatter_test.go

```diff
@@ -0,0 +1,26 @@
+package formatter
+
+import (
+	"github.com/sirupsen/logrus"
```

Comment:
Please organize the imports.
https://bongnv.com/blog/2022-03-09-why-group-imports-in-go/

Reply:
Thanks, I did not know that. I just let the IDE write the imports. I pushed a fix ordering the imports